### PR TITLE
Enhancement/improve docs for v0.0.9

### PR DESCRIPTION
**Pull request by pgarciaegido**
Hey,

Just improving the docs for our release soon. I have a lot of fields on TBD, Ill research each of them and try to come up with a good description shortly. But if you guys want to chip in, feel free 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,18 +3,27 @@
 The Crossmint SDK allows you to offer wallet-less credit card purchases on your NFT drop. It takes 10 lines of code and 5 min to integrate, and is free to use for the seller. You’ll get the sales proceeds in SOL/ETH as if the user was paying with their own wallet.
 
 Supported chains:
-* Solana
-* Polygon (private beta, contact us at sales at crossmint.io)
-* Ethereum L1 (March 2021)
+
+-   Solana
+-   Polygon (private beta, contact us at sales at crossmint.io)
+-   Ethereum L1 (end of Feb 2021)
```
Comment (alfonso-paella) [inline README.md:9]: March 2021

File: README.md [hunk 2]
```diff
@@ -3,18 +3,27 @@
...
 
 To get started:
+
 1. Fill out our form in https://www.crossmint.io/developers/onboarding (we'll review your project in under 24hr!)
 2. Follow the instructions below to integrate with your code
 
 ## Demo of the user experience:
 
+# REPLACE
```
Comment (alfonso-paella) [inline README.md:18]: Not sure if this is a placeholder? I guess it is

File: README.md [hunk 3]
```diff
@@ -25,36 +34,7 @@ yarn add @crossmint/client-sdk-react-ui
...
-```
-
-### 3. Usage
-
-Go to the main file where your Candy Machine button lives. For example, Home.tsx.
-
-There, import the Pay with `CrossMintButton`, and add it in the UI. 
-
-**Important**: be sure to test that the Crossmint button is visible even if a user didn't connect their wallet! Else, your users without wallets won't be able to use it.
+Just import the the Pay with `CrossMintButton` into your app wherever you would like. It's that simple!
```
Comment (alfonso-paella) [inline README.md:37]: Let's add back a point to ensure they test that the button shows up if a user doesn't have a crypto wallet like Phantom. A lot of ppl make this mistake where they put it in a place where it's not visible unless you have a wallet connected

File: packages/ui/react-ui/README.md [hunk 4]
```diff
@@ -2,29 +2,44 @@
...
+| collectionPhoto          | `undefined` | `false`  | This will be shown to the user during the checkout process. Preferred resolution: 200x200 px |
+| className                | `undefined` | `false`  | Use this to add custom classNames to the button                                              |
+| disabled                 | `undefined` | `false`  | Use this to specify when the button should be disabled                                       |
+| onClick                  | `undefined` | `false`  | Use this to add a custom onClick handler to the button                                       |
+| style                    | `undefined` | `false`  | Use this to add custom CSS styles to the button                                              |
+| tabIndex                 | `undefined` | `false`  | Use this to add a custom tabIndex to the button                                              |
+| theme                    | `dark`      | `false`  | Use this to specify one of our default themes. Can be `light`, `dark`                        |
+| mintTo                   | `undefined` | `false`  | TBD                                                                                          |
```
Comment (alfonso-paella) [inline packages/ui/react-ui/README.md:21]: I would leave out (undocumented) all the ones you listed as TBD, as some / most of them for now are for special partner integrations only

File: README.md (hunk 2 above)
Comment (pgarciaegido) [inline README.md:18]: I have added this as a reminder to myself to replace the old gif with a new one. Will do that shortly.

**Review by alfonso-paella — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (pgarciaegido) [reply README.md]: Actually, maybe its better to take care of it later and have a little discussion on what to show in the gif, (or if we even want a gif there)

**Review by alfonso-paella — APPROVED** [review]

**Pull request merged**